### ISSUE TITLE
mini-batch loss, how low is low enough?

### ISSUE THREAD

**Issue by kuhanw** [suriyadeepan/tf_seq2seq_wrapper#46]
Hi all,

I have been trying to duplicate the level of quality seen on the example page for the Cornell Movie dialogue and I am wondering how low have you guys driven the min-batch loss to get that type of replies? I am having trouble getting my decoder inference output to that level of comprehension. 

I am also wondering for the CornellMovieDialogue corpus. Is the input going into the model as is? When I read the dialogue some of it doesn't even seem to make sense:

('Not the hacking and gagging and spitting part.  Please.',
 "Okay... then how 'bout we try out some French cuisine.  Saturday?  Night?")

('Can we make this quick?  Roxanne Korrine and Andrew Barrett are having an incredibly horrendous public break- up on the quad.  Again.',
 "Well, I thought we'd start with pronunciation, if that's okay with you.")

as in, the answer is not referencing any information in the question.

As an additional note, I am also not exactly using the tutorials shown here, I have been piecing together tutorial scripts on seq2seq from our git for my own understanding so its always possible I have a bug in my code. Nonetheless, I do wonder about my question above.

Cheers,

Kuhan

**Comment by kuhanw** [comment]
Hey all,

As a sanity check, I just tried to reproduce the twitter results from http://suriyadeepan.github.io/2016-12-31-practical-seq2seq/, and my training loss does not seem to decrease significantly just using the out of the box code. Do I have to do some specific tuning of the hyperparameters?

My training loss in the final epochs look like : 


Model saved to disk at iteration #95000
val   loss : 3.397206

Model saved to disk at iteration #96000
val   loss : 3.424180

Model saved to disk at iteration #97000
val   loss : 3.283931

Model saved to disk at iteration #98000
val   loss : 3.356802

Model saved to disk at iteration #99000
val   loss : 3.486829

I am using the out of box  03-Twitter-chatbot.py script.

Thank You,

Kuhan

**Comment by pzhao16me** [comment]
when i at epoch 62401, the loss i got was 2.523555
